packages/odyssey-react/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Story } from "@storybook/react";
+import { action } from '@storybook/addon-actions';
+import React from "react";
+import Tooltip, { TooltipProps } from "./Tooltip";
+
+import { Button } from '../../';
```

Comment:
this import is counter-intuitive. In fact, we should be renaming all our component dirs to follow a structure like this for easier imports:

```
src
|-components
|  |-Button
|  |  |-index.tsx
|  |-Tooltip
|     |-index.tsx
```

This way your import here becomes: `import Button from '../Button'` which is much more clear

Reply:
Ill get this fixed in a follow up PR